sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPlugin.scala

```diff
@@ -261,6 +261,12 @@ object ScalaJSPlugin extends AutoPlugin {
         "scalaJSJavaSystemProperties",
         "List of arguments to pass to the Scala.js Java System.properties.",
         CTask)
+
+    val jUnitTestArgs = SettingKey[Seq[String]]("jUnitTestArgs",
+        "List of flags to pass to the JUnit framework in jUnitTest.", CSetting)
+
+    val checkJUnitTestFailuresOutput = TaskKey[Unit]("checkJUnitTestFailuresOutput",
+        "Executes tests in org.scalajs.junit.failure expecting them to fail.", DTask)
```

Comment:
Those should definitely not be in the published plugin. They are for our own build only. The should be in [`ExposedValues`](https://github.com/scala-js/scala-js/blob/v0.6.9/project/Build.scala#L45)